.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # Editor settings
 /.vscode
+.idea/
```

Comment:
I was wondering about the start vs the trailing slash: https://stackoverflow.com/a/38559600. I think we should probably use a consistent pattern to match directories, `.idea/` is different from `node_modules/` for example. I'd align them.